docs/install-machine.md

```diff
@@ -57,6 +57,21 @@ to your PATH.
         $ docker-machine -v
         machine version 0.5.0 (3e06852)
 
+## Installing bash completion scripts
```

Comment:
@leedm777 Nicely done.  I would do a few things to sort this. Put your link OUT last   --- no sense inducing them to switch channels before you've explained yourself.  Break out the functionality into a scannable list.

---> https://gist.github.com/moxiegirl/9b2dba37dc8e75f5654e

## Installing bash completion scripts

The machine repo supplies several scripts] that add features such as:

* command completion
* a function that displays the active machine in your shell prompt
* a function wrapper that adds a `docker-machine use` subcommand

The `docker-machine use function executes `eval "$(docker-machine env X)"` in the current shell.

To install the scripts, copy or link them into your `/etc/bash_completion.d` or
`/usr/local/etc/bash_completion.d` file. To enable the `docker-machine` shell prompt, add `$(__docker-machine-ps1)` to your `PS1` setting in `~/.bashrc`.

    PS1='[\u@\h \W$(__docker-machine-ps1)]\$ '

You can find additional documentation in the comments at the [top of each script](https://github.com/docker/machine/tree/master/contrib/scripts/bash).